File: src/main/scala/com/badlogic/gdx/graphics/g2d/SpriteBatch.scala
package com.badlogic.gdx.graphics.g2d

import com.badlogic.gdx.graphics.Texture

import scala.scalajs.js
import scala.scalajs.js.annotation.JSName

@JSName("gdx.graphics.g2d.SpriteBatch")
@js.native
class SpriteBatch extends js.Object {

  def begin(): Unit = js.native
  def draw(texture: Texture, x: Double, y: Double): Unit = js.native
  def end(): Unit = js.native
  def setProjectionMatrix(): Unit = js.native
}

File: libgdx/src/gdx/graphics/g2d/SpriteBatch.js
var gdx;
(function (gdx) {
    var graphics;
    (function (graphics) {
        var g2d;
        (function (g2d) {

            var Batch = gdx.graphics.g2d.Batch;
            /**
             * @JSName("gdx.graphics.g2d.SpriteBatch")
             */
            class SpriteBatch extends Batch {
                
            }
            g2d.SpriteBatch = SpriteBatch;
        })(g2d = graphics.g2d || (graphics.g2d = {}));
    })(graphics = gdx.graphics || (gdx.graphics = {}));
})(gdx || (gdx = {}));
